assets/images.json

```diff
@@ -0,0 +1,8 @@
+[
+    "plot.png",
```

Comment:
I think it would be nice to have a title + URL per image that leads to the example code (or to some future examples gallery page).